### PR TITLE
fix/variable

### PR DESCRIPTION
**Pull request by mifa43**
Cilj je da  se razdvoje linije koda. #18

### CODE REVIEW

File: src/crud.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ def insert_user(self):
 
         except psycopg2.OperationalError as e:
             raise e
+
```
Comment (milanzmitrovic) [inline src/crud.py:42]: Delete unnecessary new line character.

**Review by milanzmitrovic — CHANGES_REQUESTED** [review]
Please see my comment.

File: src/crud.py (hunk 1 above)
Comment (mifa43) [reply src/crud.py:42]: ok.

**Review by milanzmitrovic — DISMISSED** [review]
Well done.

**Review by milanzmitrovic — APPROVED** [review]

**Pull request closed without merging**